.gitignore

```diff
@@ -4,3 +4,5 @@ node_modules
 docs
 *DS_Store
 .npmrc
+lib/*.js
```

Comment:
Release scripts respect the gitingore so we can't ignore the lib. Ran into this before:
https://github.com/procore/js-sdk/commit/ad2c2f4aa204cfcb695d720d21ff184016af0e13#diff-a084b794bc0759e7a6b77810e01874f2

Reply:
Damn.. that's too bad. Makes the dev env a bit messier to work in.